docs/sources/about.md

```diff
@@ -23,18 +23,21 @@ Grafana Agent is available in three different variants:
 [Static mode Kubernetes operator]: {{< relref "./operator/" >}}
 [Flow mode]: {{< relref "./flow/" >}}
 
-## Stability
+## Release stage
 
-| Project | Stability |
+| Project | Release stage |
 | ------- | --------- |
-| Static mode | Stable |
-| Static mode Kubernetes operator | Beta |
-| Flow mode | Stable |
+| Static mode | Public preview |
+| Static mode Kubernetes operator | Experimental |
+| Flow mode | Public preview |
```

Comment:
I'm worried about this because it might seem like we're downgrading the stability of everything. 

Should we wait until 1.0 to synchronize terminology?

Reply:
Do you mean with the change from Beta to Experimental for Operator? Would it be better to just tag all as Public preview?

Reply:
I just don't think public preview is an appropriate term for the project in its current state, but neither is Generally Available (since it is not 'fully supported' by the support team).

Quoting the release life cycle document:

> To drive home my concerns, the release cycle document says this: 

> Products and features in public preview are not meant to be used in production environments, and the risks are medium/low.

I'm very concerned that saying that Static mode and Flow mode are "not meant to be used in production environments" will cause confusion and worry from our existing used.

Reply:
My $0.02 - if we don't have comprehensive support for the feature, we can't recommend that folks start using it in production...we're already seeing Cloud user confusion